### PR TITLE
LLM: add baichuan and baichuan2 to gpu pytorch model example

### PR DESCRIPTION
**Pull request by JinBridger**
## Description

Add Baichuan and Baichuan2 to GPU PyTorch model example.

### CODE REVIEW

File: python/llm/example/GPU/PyTorch-Models/Model/baichuan/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a Python environment for BigDL-LLM:
+```bash
+conda create -n llm python=3.9 # recommend to use Python 3.9
+conda activate llm
+
+# below command will install intel_extension_for_pytorch==2.0.110+xpu as default
+# you can install specific ipex/torch version for your need
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
```
Comment (plusbang) [inline python/llm/example/GPU/PyTorch-Models/Model/baichuan/README.md:19]: Refer to https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan#1-install, `transformers_stream_generator` may also necessary here.

File: python/llm/example/GPU/PyTorch-Models/Model/baichuan2/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a Python environment for BigDL-LLM:
+```bash
+conda create -n llm python=3.9 # recommend to use Python 3.9
+conda activate llm
+
+# below command will install intel_extension_for_pytorch==2.0.110+xpu as default
+# you can install specific ipex/torch version for your need
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
```
Comment (plusbang) [inline python/llm/example/GPU/PyTorch-Models/Model/baichuan2/README.md:19]: Also refer to https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan2#1-install, `transformers_stream_generator` may be required.

**Review by plusbang — COMMENTED** [review]
Add links in https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU/PyTorch-Models/Model#verified-models.

File: python/llm/example/GPU/PyTorch-Models/Model/baichuan2/README.md (hunk 2 above)
Comment (JinBridger) [reply python/llm/example/GPU/PyTorch-Models/Model/baichuan2/README.md:19]: done

File: python/llm/example/GPU/PyTorch-Models/Model/baichuan/README.md (hunk 1 above)
Comment (JinBridger) [reply python/llm/example/GPU/PyTorch-Models/Model/baichuan/README.md:19]: done

**Review by plusbang — APPROVED** [review]
LGTM

**Pull request merged**